src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,225 @@
+        >
+          <img alt="" src="/static/branding/pricing/block-green.svg" />
+        </IconButton>
+        <Divider orientation="vertical" />
+        <IconButton variant="plain">
+          <FacebookRoundedIcon />
+        </IconButton>
+        <IconButton variant="plain">
+          <GitHubIcon />
+        </IconButton>
+        <Input
+          variant="soft"
+          placeholder="Your Email"
+          type="email"
+          name="email"
+          endDecorator={(
+            <Button variant="soft" aria-label="subscribe">
+              <SendIcon />
+            </Button>
+          )}
+          sx={{ ml: 'auto', display: { xs: 'none', md: 'flex' } }}
+        />
+      </Box>
+      <Divider sx={{ my: 2 }} />
+      <Box
+        sx={{
+          display: 'flex',
+          flexDirection: { xs: 'column', md: 'row' },
+          alignItems: { md: 'flex-start' },
+          justifyContent: 'space-between',
+          flexWrap: 'wrap',
+          gap: 2,
+        }}
+      >
+        <Card
+          variant="soft"
+          size="sm"
+          sx={{
+            flexDirection: { xs: 'row', md: 'column' },
+            minWidth: { xs: '100%', md: 'auto' },
+            gap: 1,
+          }}
+        >
+          <AspectRatio
+            ratio="21/9"
+            minHeight={80}
+            sx={{ flexBasis: { xs: 200, md: 'initial' } }}
+          >
+            <img alt="" src="/static/blog/mui-product-comparison/ecosystem.png" />
+          </AspectRatio>
+          <CardContent>
+            <Typography level="body2">Intro to the MUI ecosystem</Typography>
+            <Typography level="body3" sx={{ mb: 0.5 }}>
+              MUI blog
+            </Typography>
+          </CardContent>
+        </Card>
+        <List size="sm" row wrap sx={{ flexGrow: 0, '--List-item-radius': '8px' }}>
+          <ListItem nested sx={{ width: { xs: '50%', md: 140 } }}>
+            <ListSubheader>Sitemap</ListSubheader>
+            <List>
+              <ListItem>
+                <ListItemButton>Services</ListItemButton>
+              </ListItem>
+              <ListItem>
+                <ListItemButton>Blog</ListItemButton>
+              </ListItem>
+              <ListItem>
+                <ListItemButton>Contact us</ListItemButton>
+              </ListItem>
+            </List>
+          </ListItem>
+          <ListItem nested sx={{ width: { xs: '50%', md: 180 } }}>
+            <ListSubheader>Product</ListSubheader>
+            <List sx={{ '--List-decorator-size': '32px' }}>
+              <ListItem>
+                <ListItemButton>
+                  <ListItemDecorator>
+                    <img
+                      alt=""
+                      src="/static/branding/product-core-dark.svg"
+                      width="24"
+                    />
+                  </ListItemDecorator>
+                  MUI Core
+                </ListItemButton>
+              </ListItem>
+              <ListItem>
```

Comment:
looks like separated component

Reply:
I changed from mui/joy to mui library and concept. I have added the changes for approval.